contrib/devtools/copyright_header.py

```diff
@@ -87,20 +86,20 @@ def compile_copyright_regex(copyright_style, year_style, name):
     "Bitcoin Core Developers\n",
     "the Bitcoin Core developers\n",
     "The Bitcoin developers\n",
-    "The LevelDB Authors\. All rights reserved\.\n",
-    "BitPay Inc\.\n",
-    "BitPay, Inc\.\n",
-    "University of Illinois at Urbana-Champaign\.\n",
+    r"The LevelDB Authors\. All rights reserved\.\n",
```

Comment:
I don't think simply adding `r` here is correct.
```python
>>> "The LevelDB Authors\. All rights reserved\.\n" == r"The LevelDB Authors\. All rights reserved\.\n"
False
```
A raw string will contain *literal* \.\n`` which is not the intent here.

Reply:
the problem here is that the `.` should not be escaped, not that the string is not raw !